samples/widgets/UniversalWidget/package-lock.json

```diff
@@ -1,6391 +0,0 @@
-{
-    "name": "universal-widget",
```

Comment:
It seems that you do not commit the new package-lock json here. Please use the correct node version and commit it

Reply:
added the pakage-lock.json with Node.JS v8.8.1 and NPM v5.4.2